packages/dev/s2-docs/pages/s2/index.mdx

```diff
@@ -49,7 +49,7 @@ export const title = 'Home';
   {id: 'Divider', name: 'Divider', href: 'Divider.html'},
   {id: 'DropZone', name: 'DropZone', href: 'DropZone.html'},
   {id: 'Form', name: 'Form', href: 'Form.html'},
-  {id: 'Icons', name: 'Icons', href: 'Icons.html'},
+  {id: 'Icons', name: 'Icons', href: 'icons.html'},
```

Comment:
why would this be the only one with a lowercase? seems like the file should be renamed if that's what it relies on, that way it's more predictable.

really i'd like to get rid of all the uppercases in the urls... but that's a different issue

Reply:
the convention that seems to exist currently is that component pages are capitalized (e.g. DropZone/Menu/etc) and that non component pages aren't (e.g. dnd.mdx/migrating.mdx/etc) and icons fits into the latter. The file should be renamed to be lower case as well with my last commit, forgot to force git to recognize that change since it didn't automatically